internal/utils/main.go

```diff
@@ -278,8 +278,22 @@ func MustBucketFlags(flags *pflag.FlagSet, logger *log.Entry) (path string) {
 	return
 }
 
+// MustGcsFlags gets the values of the bucket list specific flags: gcp-project and gcs-bucket
+func MustGcsFlags(flags *pflag.FlagSet, logger *EtlLogger) (bucket, credentials string) {
+	bucket, err := flags.GetString("gcs-bucket")
+	if err != nil {
+		logger.Fatal("could not get gcs bucket: ", err)
+	}
+	credentials, err = flags.GetString("gcp-credentials")
```

Comment:
❓ Does this mean we still need to pass the gcp-credentials location when it executes out of cloud composer as well? With the assumption we'll pass the default credentials?

Reply:
In the cloud composer environment, the credentials should be assumed by the default service account, so this flag should not be specified.